.github/workflows/hydroshare-resouce-to-beta.yml

```diff
@@ -0,0 +1,97 @@
+name: MinIO Sync: Prod to Beta
+
+on:
+  workflow_dispatch:
+    inputs:
+      resource_ids:
+        description: 'Comma-separated list of resource IDs to check and sync'
+        required: true
+
+jobs:
+  check-and-sync:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v4
+
+    - name: Install MinIO CLI (mc)
+      run: |
+        curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
+        chmod +x mc
+        sudo mv mc /usr/local/bin/mc
+
+    - name: Create MinIO Config File
+      env:
```

Comment:
You can collapse this run and the next run into just setting the environment variables with https://min.io/docs/minio/linux/reference/minio-mc/minio-client-settings.html#envvar.MC_HOST_-ALIAS

Reply:
The rest of it looks good and can should run successfully, given correct secret values.